project.clj

```diff
@@ -5,8 +5,9 @@
             :url "http://opensource.org/licenses/MIT"}
   :scm {:name "git"
         :url "http://github.com/dm3/clojure.java-time"}
-  :dependencies [[clj-tuple "0.2.2"]]
-  :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.0"]
+  :dependencies [[org.clojure/clojure "1.10.1" :scope "provided"]
```

Comment:
Not sure how I feel about this. I guess if we bump to 0.4.0 this should be fine.